src/test/java/com/microsoft/sqlserver/jdbc/TestUtils.java

```diff
@@ -274,6 +274,18 @@ public static void dropTableIfExists(String tableName, java.sql.Statement stmt)
         dropObjectIfExists(tableName, "U", stmt);
     }
 
+    /**
+     * Deletes the contents of a table.
+     * @param con
+     * @param tableName
+     * @throws SQLException
+     */
+    public static void clearTable(java.sql.Connection con, String tableName) throws SQLException {
```

Comment:
not a big deal...but just "Connection" would do we already import java.sql.Connection